birdhouse/optional-components/all-public-access/all-public-access-magpie-permission.cfg

```diff
@@ -0,0 +1,235 @@
+permissions:
+  - service: catalog
+    permission: getcapabilities
+    user: authtest
+    action: create
+
+  - service: catalog
+    permission: describeprocess
+    user: authtest
+    action: create
+
+  - service: catalog
+    permission: execute
+    user: authtest
+    action: create
+
+
+  - service: finch
+    permission: getcapabilities
+    user: authtest
+    action: create
+
+  - service: finch
+    permission: describeprocess
+    user: authtest
+    action: create
+
+  - service: finch
+    permission: execute
+    user: authtest
+    action: create
+
+
+  - service: flyingpigeon
+    permission: getcapabilities
+    user: authtest
+    action: create
+
+  - service: flyingpigeon
+    permission: describeprocess
+    user: authtest
+    action: create
+
+  - service: flyingpigeon
+    permission: execute
+    user: authtest
+    action: create
+
+
+  - service: hummingbird
+    permission: getcapabilities
+    user: authtest
+    action: create
+
+  - service: hummingbird
+    permission: describeprocess
+    user: authtest
+    action: create
+
+  - service: hummingbird
+    permission: execute
+    user: authtest
+    action: create
+
+
+  - service: lb_flyingpigeon
```

Comment:
I would remove all mentions of lb_flyingpigeon, no tests are targetting it.  It is an extra config that is not in this repo.  In fact it's probably a mistake to have left it in `birdhouse/config/magpie/providers.cfg.template`, sorry.  I'll fix this in a different PR I need to sync this with our "private config" that will restore it.